### PR TITLE
Add person.rb

### PR DESCRIPTION
**Pull request by ecekpo**
--Added--
- Changes effected on Student.rb file
- Changes effected on Teachers.rb file
- Changes on Readme file

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super()
```
Comment (cynthiainga) [inline student.rb]: - [ ] Please kindly pass `age, name, and parent_permission` arguments in initialize then call it in `super` 🙃 🙃

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
```
Comment (cynthiainga) [inline teacher.rb]: - [ ]  Please kindly pass `age, name, and parent_permission` arguments in initialize then call it in `super` 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @ecekpo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
-  [OPTIONAL] The link to your license is broken. Please fix it to make your readme more professional and descriptive. I would like to strongly recommend you 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (rloterh) [reply student.rb]: well resolved! ✔️

File: teacher.rb (hunk 2 above)
Comment (rloterh) [reply teacher.rb]: well resolved! ✔️

File: student.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require './person'
```
Comment (rloterh) [inline student.rb:1]: - [ ] OPTIONAL
By convention, it's a good practice to use `require_relative` classes in the same source and use `require` for external modules or packages. Kindly ensure to improve upon this aspect.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Cookey,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾  Congratulations!  🎉💯🌟 

## Highlights 👍
- Nice job implemented required changes

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**